__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -11,6 +11,7 @@ const chatSchema = new ChatSchema({
   createdAt: 'createdAt',
   updatedAt: 'updatedAt',
   rawJson: {},
+  attachmentJson: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` field is initialized as an empty array. However, in the database layer, it's likely that this field will be stored as a string (since SQLite doesn't support arrays). Therefore, it would be better to initialize it as an empty string or null.
```diff
-  attachmentJson: [],
+  attachmentJson: '',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->